client/base.jsx

```diff
@@ -0,0 +1,43 @@
+
+/**
+ * Module dependencies.
+ */
+
+var React = require('react');
+
+module.exports = React.createClass({
+	render: function() {
+
+		var googleMapsScript =
+		    'function initialize() { dispatcher.dispatch(\'GOOGLE_MAPS_READY\'); } ' +
+			'google.maps.event.addDomListener(window, \'load\', initialize);';
+
+		var googleMapsBootstrap = '';
+
+		if (this.props.useGoogleMaps) {
+			googleMapsBootstrap = [
```

Comment:
You mean, why the array syntax? Yeah, I guess... to compose react elements like this, you have to have each piece be a valid react node. An array of react nodes is a react fragment (which is itself a react node). 

At the bottom of this page they have a list of formal definitions of things: http://facebook.github.io/react/docs/glossary.html